spec/models/card_spec.rb

```diff
@@ -1,10 +1,15 @@
 require "rails_helper"
+require 'database_cleaner'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.